indico/modules/events/models/events.py

```diff
@@ -843,7 +844,7 @@ def log(self, realm, kind, module, summary, user=None, type_='simple', data=None
         if self.__logging_disabled:
             return
         entry = EventLogEntry(user=user, realm=realm, kind=kind, module=module, type=type_, summary=summary,
-                              data=data or {})
+                              data=data or {}, internal_data=internal_data or {})
```

Comment:
```suggestion
                              data=(data or {}), internal_data=(internal_data or {}))
```

No technical difference but kwarg values with spaces look very weird without parentheses.